src/utils/forest-url-getter.js

```diff
@@ -0,0 +1 @@
+module.exports = () => process.env.FOREST_URL || 'https://api.forestadmin.com';
```

Comment:
[THINKING] If we use the `.addValue` as pointed out above, we might want to skip this file